EN/BattleRes/talk/script/chr012/talk_chr012_50_03.txt

```diff
@@ -34,41 +35,41 @@ Kohaku, I've found one.
 
 /*-auto-*/\\@VOICE_PLAY@<0, "01His/His05_0041", 100>
 Hisui
-I believe this may be the person in question who 
-has been running amok destroying things in town.
+I believe this may be the person in question who
+has been running amok in town.
 \l
 
 \\@CHARA_IN@< 3, 1, 1, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "01His/His05_0042", 100>
 Hisui
-Let's just say it. That's the Knight King.
+Let's just say it. That's the King of Knights.
```

Comment:
Comun wanted to go with Knight King just for fun (it's the more correct translation) so I think we'll probably keep it to that

Reply:
I much prefer King of Knights personally, but if that's what Conun wants, I can revert it.